command2.bat

```diff
@@ -245,4 +245,4 @@ echo Melody Script Applied. This PC will reboot.
 xcopy "MelodyScript.IntegratedTools\EmptyStandbyList\*" %windir% /y /e /h /c /i
 start /wait powershell.exe -command %windir%\install.ps1
 shutdown /r /f /t 0
-:--------------------------------------
```

Comment:
i will accept first
```
reg add "HKLM\SYSTEM\CurrentControlSet\Enum\%%i\Device Parameters\Interrupt Management\MessageSignaledInterruptProperties" /v "MSISupported" /t REG_DWORD /d "1" /f 
reg add "HKLM\SYSTEM\CurrentControlSet\Enum\%%i\Device Parameters\Interrupt Management\MessageSignaledInterruptProperties" /v "MSISupported" /t REG_DWORD /d "0" /f
```
but this part 
```
netsh.exe interface tcp set heuristics disabled >nul
```
no because this command to keep persistent the autotuning configuration from global tcp-ip setting.

removing that the autotuning value is reverted in next reboot.

Reply:
this autotuning setting is ok gigabit speeds to get maximum speed of internet.

Reply:
oh ok thanks for that information:)